### PR TITLE
fixed tutorial code of py_matcher

### PR DESCRIPTION
**Pull request by atinfinity**
resolves https://github.com/opencv/opencv/issues/13615

### CODE REVIEW

File: doc/py_tutorials/py_feature2d/py_matcher/py_matcher.markdown [hunk 1]
```diff
@@ -53,8 +53,8 @@ import numpy as np
 import cv2 as cv
 import matplotlib.pyplot as plt
 
-img1 = cv.imread('box.png',0)          # queryImage
-img2 = cv.imread('box_in_scene.png',0) # trainImage
+img1 = cv.imread('box.png',cv.IMREAD_UNCHANGED)          # queryImage
+img2 = cv.imread('box_in_scene.png',cv.IMREAD_UNCHANGED) # trainImage
```
Comment (alalek) [inline doc/py_tutorials/py_feature2d/py_matcher/py_matcher.markdown]: 0 is `IMREAD_GRAYSCALE`

**Comment by alalek** [comment]
Thnak you!

As a "bugfix" this patch should [go](https://github.com/opencv/opencv/wiki/How_to_contribute#faq) into [3.4 branch](https://github.com/opencv/opencv/wiki/Branches) first. We will merge changes from 3.4 into master regularly (weekly/bi-weekly).

So, please:
- change "base" branch of this PR: master => 3.4 (use "Edit" button near PR title)
- rebase your commits from master onto 3.4 branch. For example:
    `git rebase -i --onto upstream/3.4 upstream/master`
    (check list of your commits, save and quit (Esc + "wq" + Enter)
    where `upstream` is configured by following [this GitHub guide](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and fetched (`git fetch upstream`).
- push rebased commits into source branch of your fork (with `--force` option)

Note: no needs to re-open PR, apply changes "inplace".

File: doc/py_tutorials/py_feature2d/py_matcher/py_matcher.markdown (hunk 1 above)
Comment (atinfinity) [reply doc/py_tutorials/py_feature2d/py_matcher/py_matcher.markdown]: Thank you for your review. I fixed the mode of imread.

**Comment by atinfinity** [comment]
@alalek 

> As a "bugfix" this patch should go into 3.4 branch first. We will merge changes from 3.4 into master regularly (weekly/bi-weekly).

I changed target branch from master to 3.4.
But, CI job failed. I'll investigate.

**Pull request merged**